diff --git a/src/main/utils/constants.ts b/src/main/utils/constants.ts
index 9513da51..0f32155e 100644
--- a/src/main/utils/constants.ts
+++ b/src/main/utils/constants.ts
@@ -22,7 +22,7 @@ import { join } from "path";
 export const DATA_DIR = process.env.VENCORD_USER_DATA_DIR ?? (
     process.env.DISCORD_USER_DATA_DIR
         ? join(process.env.DISCORD_USER_DATA_DIR, "..", "VencordData")
-        : join(app.getPath("userData"), "..", "Vencord")
+        : join(app.getPath("home"), ".config", "Vencord")
 );
 export const SETTINGS_DIR = join(DATA_DIR, "settings");
 export const THEMES_DIR = join(DATA_DIR, "themes");
